db/migrate/20221203190040_create_posts.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :text
+      t.integer :comments_counter
+      t.integer :likes_counter
+      t.references :user, null: false, foreign_key: true
```

Comment:
- [ ] Taking into consideration the given ERD, the reference name should be `author` instead of `user`, hence creating a migration query should be similar to:

```ruby
...
  def change
    add_reference :posts, :author, references: :users, null: false, foreign_key: {to_table: :users}
  end
...